backend/news/tasks.py

```diff
@@ -0,0 +1,11 @@
+# backend.news/tasks.py
+from celery import shared_task
+from backend.celery_script import ArticleScraper
+from datetime import datetime
+
+
+@shared_task
+def run_scraper():
+    start_url = 'https://www.thenews.com.pk/'
```

Comment:
This should be a constant inside the ArticleScraper class because this class will only work for this news website.